From 734e60f934d8f94f91b61d27b4e5118b4ac73220 Mon Sep 17 00:00:00 2001
From: Andra-Teodora Ilie <andra.ilie@nxp.com>
Date: Tue, 13 Apr 2021 16:31:32 +0300
Subject: [PATCH 34/42] s32: Remove duplicated 'run loadimage' command

When 'run mmcboot' is executed, the kernel image
is already loaded from bootcmd.

Upstream-Status: Pending

Issue: ALB-6884
Signed-off-by: Andra-Teodora Ilie <andra.ilie@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32.h b/include/configs/s32.h
index 4db1d9750e..e33435dd69 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -489,7 +489,7 @@
 		"${boot_mtd} ${loadaddr} ${ramdisk_addr} ${fdt_addr}\0" \
 	"mmcboot=echo Booting from mmc ...; " \
 		"run mmcargs; " \
-		"run loadimage; if run loadfdt; then " \
+		"if run loadfdt; then " \
 			"${boot_mtd} ${loadaddr} - ${fdt_addr}; " \
 		"else " \
 			"echo WARN: Cannot load the DT; " \
-- 
2.25.1

